x-pack/filebeat/input/awscloudwatch/input.go

```diff
@@ -149,9 +149,9 @@ func (in *cloudwatchInput) Run(inputContext v2.Context, pipeline beat.Pipeline)
 		return err
 	}
 
-	log.Debugf("Config latency = %f", cwPoller.config.Latency)
-	log.Debugf("Config scan_frequency = %f", cwPoller.config.ScanFrequency)
-	log.Debugf("Config api_sleep = %f", cwPoller.config.APISleep)
+	log.Debugf("Config latency = %d", cwPoller.config.Latency)
+	log.Debugf("Config scan_frequency = %d", cwPoller.config.ScanFrequency)
+	log.Debugf("Config api_sleep = %d", cwPoller.config.APISleep)
```

Comment:
This one can be ignored, but printing a `time.Duration` as integer makes it hard to know the actual value. It would be better to convert it to string. Changing the verb to `%s` will do the trick. We could call `.String()` directly, but I think it's unnecessary.

```suggestion
	log.Debugf("Config latency = %s", cwPoller.config.Latency)
	log.Debugf("Config scan_frequency = %s", cwPoller.config.ScanFrequency)
	log.Debugf("Config api_sleep = %s", cwPoller.config.APISleep)
```